connections.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$id": "https://github.com/getdock/schemas/connections.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "User social connections",
+  "title": "User network connections",
+  "definitions": {
+    "connection": {
+      "type": "object",
+      "properties": {
+        "firstName": {
+          "type": "string"
+        },
+        "lastName": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string",
+          "format": "email"
+        },
+        "company": {
+          "type": "string"
+        },
+        "position": {
+          "type": "string"
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "uniqueItems": true
+          }
+        },
+        "connectedOn": {
+          "type": "number",
+          "format": "timestamp"
+        },
+        "source": {
+          "type": "string",
+          "description": "Optional field that denotes where this connection is coming from"
+        }
+      },
+      "required": [
+        "first_name",
+        "last_name",
+        "email"
+      ]
+    }
+  },
+  "type": "object",
+  "properties": {
```

Comment:
What is the purpose of `properties` object? What could it do apart from repeating the content of `definitions`?

Reply:
According to json schema specification, `properties` object defines set of fields in a given object. It is different from definitions, because definitions let you define specific item once and reuse it in different places by referencing to it. In this particular example it is not that obvious (because there is only one definition that is used once) however, it makes sense to separate connection object definition from overall package structure.